generators/aws-swarm-tokens/templates/aws-swarm-tokens/index.ts

```diff
@@ -0,0 +1,51 @@
+import { remote } from "@pulumi/command";
+import { secret } from "@pulumi/pulumi";
+
+import { getConfig } from "./config";
+
+export = async () => {
+
+  const config = await getConfig();
+
+  if (!config.leaderIp) {
+    throw new Error("leaderIp is required but was not provided or found");
+  }
+
+  const conn = {
+    host: config.leaderIp,
+    user: "ec2-user",
```

Comment:
Make this a variable, with `ec2-user` as the default value. This is an easy way to add extensibility to this project.